tests/unit/amo/pages/TestAddonInfo.js

```diff
@@ -0,0 +1,392 @@
+import * as React from 'react';
+
+import AddonInfo, {
+  ADDON_INFO_TYPE_EULA,
+  ADDON_INFO_TYPE_PRIVACY_POLICY,
+  AddonInfoBase,
+  extractId,
+} from 'amo/pages/AddonInfo';
+import AddonSummaryCard from 'amo/components/AddonSummaryCard';
+import {
+  createInternalAddon,
+  fetchAddon,
+  fetchAddonInfo,
+  loadAddonInfo,
+  loadAddonResults,
+} from 'core/reducers/addons';
+import {
+  createFakeLocation,
+  createStubErrorHandler,
+  dispatchClientMetadata,
+  fakeAddon,
+  fakeAddonInfo,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import ErrorList from 'ui/components/ErrorList';
+import LoadingText from 'ui/components/LoadingText';
+
+describe(__filename, () => {
+  let store;
+
+  beforeEach(() => {
+    store = dispatchClientMetadata().store;
+  });
+
+  const getProps = ({
+    location = createFakeLocation(),
+    params,
+    ...customProps
+  } = {}) => {
+    return {
+      i18n: fakeI18n(),
+      infoType: ADDON_INFO_TYPE_PRIVACY_POLICY,
+      location,
+      match: {
+        params: {
+          slug: fakeAddon.slug,
+          ...params,
+        },
+      },
+      store,
+      ...customProps,
+    };
+  };
+
+  const render = ({ ...customProps } = {}) => {
+    const props = getProps(customProps);
+
+    return shallowUntilTarget(<AddonInfo {...props} />, AddonInfoBase);
+  };
+
+  const loadAddon = (addon = fakeAddon) => {
```

Comment:
A pattern we tried to spread everywhere is to use `_{actionCreator}`, like `_loadAddonResults` in your case. I see that you are loading only one add-on, but maybe that could still be done:

```js
const _loadAddonResults = (addons = [fakeAddon]) =>
```

Usage:

```js
_loadAddonResults([addon]);
```